contributions/catalog/Dockerfile

```diff
@@ -16,10 +16,14 @@ RUN apk --update add python3 py3-pip openssl ca-certificates py3-openssl && \
 
 WORKDIR /usr/src/app/catalog
 
-ENV MONGO_CONTRIBUTION_URL="" \
-    API_LOC="." \
-    FLASK_APP="catalog_rest_service" \
-    FLASK_ENV="development" \
-    CATALOG_URL_PREFIX=""
+ENV CONTRIBUTION_BUILDING_BLOCK_URL='' \
```

Comment:
I have not tested the Catalog, but we can get to this for Catalog deployment. Thanks.